kiali.go

```diff
@@ -268,7 +265,7 @@ func updateBaseURL(webRootPath string) {
 	html := string(b)
 
 	searchStr := `<base href="/"`
-	newStr := `<base href="` + webRootPath + `/"`
+	newStr := `<base href="` + webRootPath + `"`
```

Comment:
Oh, I see... you are removing the trailing slash here. We need to think about this some more, because I am wondering why the original developer added that  trailing slash here. Clearly it was needed.

We just need to confirm the normal use cases work in addition to the reverse proxy before merging this PR.

Did you run with web_root as `/` and `/kiali` and see if that works?

Reply:
with `/` i had tested it, let me test it with `/kiali` and  will let you know